test/functional/example_test.py

```diff
@@ -78,7 +78,10 @@ def set_test_params(self):
         """Override test parameters for your individual test.
 
         This method must be overridden and num_nodes must be explicitly set."""
-        self.setup_clean_chain = True
+        # By default the test loads a pre-mined chain of 200 blocks from cache.
+        # Set use_chached_chain to False to skip this and start from the Genesis
+        # block.
+        self.use_cached_chain = False
```

Comment:
Noob confession: when I first started tinkering with example_test.py, I struggled a lot because I didn’t quite understand what a “clean chain” meant and wasn't aware that coinbases needed to mature (I was trying to create transactions with `sendtoaddress` and getting "insufficient funds"). Maybe comments/docs here or elsewhere could help?

Reply:
Hm, I get your point but I think explaining coinbase maturity, a very basic part of the protocol, here in the example test or in the functional test README, feels out of place to me.